blocks/columns/columns.js

```diff
@@ -15,4 +15,22 @@ export default function decorate(block) {
       }
     });
   });
+
+  if (block.classList.contains('collections')) {
+    block.parentElement.classList.add('column-collections-parent');
+    // query anchor tags in the block and move the picture into the anchor
+    const anchors = [...block.querySelectorAll('a')];
+    anchors.forEach((anchor) => {
+      const h2 = document.createElement('h2');
+      h2.classList.add('columns-img-header');
+      h2.innerHTML = anchor.textContent.toUpperCase();
+      anchor.text = '';
```

Comment:
better use the well-known `textContent`
```suggestion
      anchor.textContent = '';
```